### DISCUSSION TITLE
Merging Serial Cuts

### DISCUSSION THREAD

**Discussion by rosedever** [IBEXImagingCommunity/ibex_imaging_knowledge_base#191] [Q&A] [answered]
I am currently working on a project that involves using antibodies on serial cuts of NHP FFPE sections. Some of the antibodies that I am using cannot be used on the same slide, so we have resulted to using serial cuts. My problem is now that I have acquired images of the antibodies on the different slides, I would ideally like to merge them into a single image by overlaying the images, but I am struggling to complete this in Imaris. Do you have recommendations on how to overlay of images in Imaris?
Thank you!

**Comment by zivy** [comment, marked as answer]
Hello @rosedever,

Generally speaking, the way you are describing the problem makes it ill posed (many plausible solutions). We can however get an approximate solution if we make the following assumption, the section thickness is known from the microtome settings, let's say 10&mu;m.

The following imaris workflow and an (ab)use of the [XTRegisterSameChannel extension](https://github.com/niaid/imaris_extensions) may work:

0. Save a backup of the images (we're going to modify the data below, so keep the originals somewhere safe).
1. Open each slice in imaris and make sure the voxel size for Z matches the known section thickness (Edit->Image properties). If not, change it and save, the values are editable.

2. Using the XTRegisterSameChannel register consecutive slices to each other. Pseudo-code description of steps that you will need to do:
```
     for i in range(num_slices):
        output.ims = XTRegisterSameChannel(fixed_image=slice_i, moving_image=slice_i+1)
        open(output.ims) in imaris and delete the channel corresponding to slice_i (Edit->delete channels), save/overwrite as slice_i+1
```
4. Open slice1 in imaris, edit->add slices add slice2, repeat slice by slice.

Hopefully this isn't too confusing.

**Reply by rosedever** [reply]
Hi!

Thank you so much!! I was able to overlap with the extension!

**Reply by zivy** [reply]
Congrats. Best of luck with your research.
